solutions/secure/variables.tf

```diff
@@ -120,10 +114,9 @@ variable "existing_kms_key_crn" {
   default     = null
 }
 
-variable "existing_kms_instance_guid" {
-  description = "The GUID of an existing Hyper Protect or Key Protect instance in the same account as the Elasticsearch database instance. Always used to create an authorization policy and if 'existing_kms_key_crn' is not specified also used to create a KMS root key"
+variable "existing_kms_instance_crn" {
+  description = "The CRN of an existing Hyper Protect or Key Protect instance in the same account as the Elasticsearch database instance. Always used to create an authorization policy and if 'existing_kms_key_crn' is not specified also used to create a KMS root key"
   type        = string
-  default     = null
```

Comment:
default this back to `null` because its not always required (for example if user passes existing key)

Reply:
Is that true? It is passed to the elastic search instance. It would depend if that is happy to take null...

Reply:
Where it is used for the authorization policy depending on the value of skip_auth_policy which we allow the consumer to set.

Reply:
Ah... so it could be null if skipping auth policy and using an existing key. I see. Sorry to be slow.

Reply:
addressed